packages/api-contract/src/types.ts

```diff
@@ -55,8 +55,8 @@ export interface InterfaceContractCalls {
 
 export interface ContractCallOutcome {
   debugMessage: Text;
-  gasConsumed: u64;
-  gasRequired: u64;
+  gasConsumed: bigint | string | number | BN | WeightV2;
```

Comment:
i think here it should be `u64 | WeightV2`

Reply:
the `ContractCallOutcome` interface represents what is returned from the runtime call that executes on-chain. `gasConsumed` and `gasRequired` can't ever be returned as string.

on the other hand `ContractOptions` represents what can be passed in the runtime call by users. i suppose the api converts it before sending and it's more convenient to have looser typing here.

take a look at how it is on master. they are not the same.

Reply:
Thank you for clarify! I have changed as your suggest